packages/components/src/components/contract-card/contract-card-items/contract-update-form.tsx

```diff
@@ -25,7 +25,7 @@ export type TContractUpdateFormProps = {
     getCardLabels: TGetCardLables;
     onMouseLeave: () => void;
     removeToast: (toast_id: string) => void;
-    setCurrentFocus: (name: string) => void;
+    setCurrentFocus: (name: string | null) => void;
     status: string;
     toggleDialog: (e: any) => void; // This function accomodates events for various HTML elements, which have no overlap, so typing it to any
```

Comment:
```suggestion
    toggleDialog: (e: React.MouseEvent<HTMLButtonElement>) => void;
```
the comment is wrong because this function gets the event fired on a specific button HTML element.